src/porepy/examples/flow_benchmark_2d_case_1.py

```diff
@@ -12,43 +12,36 @@
 
 """
 
-from typing import Callable, Union
+from dataclasses import dataclass
+from typing import Callable, ClassVar, Union
 
 import numpy as np
 
 import porepy as pp
 from porepy.applications.discretizations.flux_discretization import FluxDiscretization
+from porepy.compositional.materials import SolidConstants, _HashableDict
 from porepy.models.constitutive_laws import DimensionDependentPermeability
 
 
-class FractureSolidConstants(pp.SolidConstants):
+@dataclass(frozen=True, kw_only=True)
+class FractureSolidConstants(SolidConstants):
```

Comment:
I think there is no part in the tutorial explicitly for material constants.
Since the general `fluid` itself needs one, could we put it there? This is probably a new user group issue.

Reply:
Sounds good. Could you create and label the issue, please?

Reply:
Done!